src/dmd/dinterpret.d

```diff
@@ -4759,7 +4736,7 @@ public:
             else
                 break;
         }
-        const cmpop = nott ? reverseRelation(ex.op) : ex.op;
+        const TOK cmpop = nott ? reverseRelation(ex.op) : ex.op;
```

Comment:
This change in unnecessary.

Reply:
Actually, it is, if Expression.op is const.